### DISCUSSION TITLE
Ability to backup/sync configurations?

### DISCUSSION THREAD

**Discussion by dhcpserver** [AdguardTeam/AdGuardHome#3703] [General]
Love the product.  I'm setting it up for a bunch of non-profits using a couple of cloud-based providers behind a firewall.

Is there any way to either backup configurations so when I build new instances they mirror or even better, is there a way to synchronize configurations so if I make a change to the block lists or add a specific domain to block, it can sync across the different AGH instances?

I know I'm probably pushing it beyond the original design intent, but it's such a slick product and for some non-profits out there, an affordable way to prevent their non-tech-savvy users from going somewhere bad.  I've done this with PiHole using Cloudsync and other replication methods, just curious to see if AGH could do it.

If not, maybe it could be a feature request?

Thanks, all!

**Comment by timkgh** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/573
